joomla/joomla-cms
# Fix deprecated warning in contenthistory

Pull Request for Issue # .

### Summary of Changes
Fix deprecated warning in contenthistory. It is caused by the null value for example in checked_out_time.

![grafik](https://user-images.githubusercontent.com/1035262/227803397-5fcfec53-2b82-4521-8bf1-aeb6e9fd8c17.png)



### Testing Instructions
See screenshot. Oben the contenthistory of an article and open one if the versions. 


### Actual result BEFORE applying this Pull Request
See screenshot

### Expected result AFTER applying this Pull Request
No warning


### Link to documentations
Please select:
